lineman/app/models/draftable_model.coffee

```diff
@@ -0,0 +1,15 @@
+angular.module('loomioApp').factory 'DraftableModel', (BaseModel) ->
+  class DraftableModel extends BaseModel
```

Comment:
This is the first attempt at generalizing draftable objects.

idea:
- The model defines the name of the method to call for it's draft model (so, discussion calls group(), motion calls discussion(), etc.)
- Based on that, we update the payload of that object, with the model name, ie

```
# a discussion draft
payload: {
  motion: { ...update here for a new proposal draft... }
  comment: { ...update here for a new comment draft... }
}
```

```
# a group draft
payload: {
  discussion: { ...update here for a new thread... }
}
```